src/OmniSharp.Abstractions/Models/v1/FileOperation/FileModificationType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace OmniSharp.Models
+{
+    public enum FileModificationType
```

Comment:
there is already a public `FileChangeType` enum for the watcher, isn't it a bit confusing to add `FileModificationType` too?

Reply:
This is pretty different though. They're not really interchangeable.